src/types/main.ts

```diff
@@ -5,6 +5,12 @@ export interface Location {
   longitude: number;
 }
 
+export interface UserLocation {
```

Comment:
Is this new interface really necessary? I'm not 100%, but I feel that this info could be represented with already existing types (maybe just a location?)